### DISCUSSION TITLE
UX: too many clicks to boost

### DISCUSSION THREAD

**Discussion by Geobert** [cheeaun/phanpy#588] [Ideas]
We need 3 clicks in order to boost a toot and it feels too much to me. Am I the only one?

**Comment by Fastidious** [comment]
I have gotten used to it. When I want to boost, those 3 taps (I mostly use Phanpy while on mobile) happen in a second.

**Comment by Crazypedia** [comment]
yeah, it's a lot, and I find myself hitting other buttons when they're so small on the mobile UI
